Source/WebCore/page/RemoteFrame.cpp

```diff
@@ -105,4 +105,9 @@ void RemoteFrame::frameDetached()
     m_client->frameDetached();
 }
 
+String RemoteFrame::renderTreeDump(int baseIndent)
```

Comment:
size_t.  This doesn't need to be able to support negative values.  Same for all the places where indentation is passed around.

Reply:
Yes, I used an int here mainly because `m_indent` is stored as an int and setIndent() takes in an int, but I can change that.